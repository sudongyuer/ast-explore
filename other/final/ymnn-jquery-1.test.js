import pluginTester from 'babel-plugin-tester'
import ymnnJquery from './ymnn-jquery-1'

pluginTester({
  plugin: ymnnJquery,
  tests: [
    {code: 'foo.hide();', snapshot: false},
    {code: `$(el).hide();`, snapshot: true},
  ],
})
