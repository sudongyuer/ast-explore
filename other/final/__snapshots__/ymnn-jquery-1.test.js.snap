// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2. ymnn-jquery 1`] = `
"
$(el).hide();

      ↓ ↓ ↓ ↓ ↓ ↓

el.style.display = 'none';
"
`;
